lib/pos_mempool.go

```diff
@@ -190,6 +203,16 @@ func (mp *PosMempool) Start() error {
 		return errors.New("PosMempool.Start: PosMempool not initialized")
 	}
 
+	// Create the transaction register, the ledger, and the nonce tracker,
+	mp.txnRegister = NewTransactionRegister(mp.globalParams)
+	mp.feeEstimator = NewPoSFeeEstimator()
+	// TODO: parameterize num mempool blocks and num past blocks. Also, how to pass in blocks.
+	if err := mp.feeEstimator.Init(mp.txnRegister, 1, []*MsgDeSoBlock{}, 60, mp.globalParams); err != nil {
+		return errors.Wrapf(err, "PosMempool.Start: Problem initializing fee estimator")
+	}
```

Comment:
Any reason we call `mp.feeEstimator.Init(...)` here in the `PosMempool.Start(...)` if it's already called in `PosMempool.Init(...)` above? The feeEstimator should already be initialized with the right blocks before the `PosMempool.Start(...)` function is called
```suggestion
```